EIPS/eip-1271.md

```diff
@@ -37,8 +37,8 @@ contract ERC1271 {
 
   /**
    * @dev Should return whether the signature provided is valid for the provided data
-   * @param _data Arbitrary length data signed on the behalf of address(this)
-   * @param _signature Signature byte array associated with _data
+   * @param _hash hash of the data that is signed on the behalf of address(this)
+   * @param _signature Signature byte array of the data
```

Comment:
Changing from "associated with the _data" to "of the data" seems like a normative change as those two statements have different meanings.  Is it established elsewhere in this EIP that the signature is specifically of the data itself and not just associated with it?

Reply:
You are correct. I changed it back to be "associated with" but with the _hash.